apps/api-site-test/test/integration/get-current-site.spec.ts

```diff
@@ -129,54 +129,7 @@ describe('Get Current Site', () => {
 
       const body: IGetSiteApiResponse = response.body
       expect(body.name).toEqual('Test Site 3')
-      expect(body.pages).toEqual(pagesAfter)
-    })
-
-    it('returns current when requester is admin site for subdomain', async () => {
```

Comment:
I removed auth from `get-current-site`, since we only use the endpoint for serving the live site. It simplifies the code a bit and makes it faster!